apps/app/src/containers/DocumentationPhrase.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { x } from "@xstyled/styled-components";
+import { Link } from "@argos-ci/app/src/components";
+
+export const DocumentationPhrase = (props) => {
+  return (
+    <x.span {...props}>
+      {" "}
+      Read our documentation for more information about{" "}
+      <Link target="_blank" href="https://docs.argos-ci.com">
+        installing
+      </Link>{" "}
+      Argos and{" "}
+      <Link target="_blank" href="https://docs.argos-ci.com/usage">
+        using it
+      </Link>
+      .
```

Comment:
Mettre le lien sur la documentation, qui va s'amuser à cliquer sur installing puis sur using it.